packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -436,6 +436,46 @@ class SliverGrid extends SliverMultiBoxAdaptorWidget {
     @required this.gridDelegate,
   }) : super(key: key, delegate: delegate);
 
+  /// Creates a sliver that places multiple box children in a two dimensional
+  /// arrangement with a fixed number of tiles in the cross axis.
```

Comment:
we try to avoid using the same words as in the name of the member being documented, because if someone is reading the docs, then the name has already failed them. :-)